file-formats/fugr/type/zif_fb_aff_reps_v1.intf.abap

```diff
@@ -0,0 +1,41 @@
+INTERFACE zif_fb_aff_reps_v1
```

Comment:
Can we use this interface for all report sources (independent of function modules)?